assembl/static2/js/app/components/common/attachFileForm.jsx

```diff
@@ -13,7 +13,8 @@ type AttachFileFormState = {
   file: File | null
 };
 
-class AttachFileForm extends React.Component<*, AttachFileFormProps, AttachFileFormState> {
+class AttachFileForm extends React.Component<AttachFileFormProps, AttachFileFormState> {
+  static defaultProps: *;
```

Comment:
Was this really needed? I guess its flow-upgrade that put this there? I think you can remove all `static defaultProps: *` that was added. I think we really meant void instead of any there.

Reply:
Yes, this code change has been made by flow-upgrade. I did not try to optimize what it did, my knowledge of flow subtilities is still limited. What "any" are you talking about?